web/cobrands/fixamingata/base.scss

```diff
@@ -231,6 +235,18 @@ $grid-breakpoint-sm: $mysoc-footer-breakpoint-sm;
     padding-top: 0;
 }
 
+#geolocate_link {
+    margin-top: 2em !important;
+}
+
+.mobile #geolocate_link {
+    margin-top: 15px !important;
+}
+
 #pc-hint {
     display: none;
 }
+
+.sticky-sidebar li:last-child {
```

Comment:
Think this will be pulled in by a rebase from core.

Reply:
Please remove it, thanks!